docs/docs/fargate-tutorials/prisma.md

```diff
@@ -79,10 +79,12 @@ cd infra
 npm i
 ```
 
-3. Modify the [`.env file`](https://www.prisma.io/docs/guides/development-environment/environment-variables) that Prisma expects with the connection parameters provided on db creation. In this case:
+3. Modify the [`.env file`](https://www.prisma.io/docs/guides/development-environment/environment-variables) that Prisma expects with the connection parameters provided on db creation. Also, you'll need to add your [Github personal acccess token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for the `ecr_builder` module to be able to do the pull. Also, if you're going to deploy a codebase other than the default one, set the `REPO_URI` variable. In this case:
```

Comment:
Nit. Don't use `Also` as the first word in two sentences in a row. You can probably just drop it from the first of the two and it'll read better (especially since it seems the github access token is non-optional).

```suggestion
3. Modify the [`.env file`](https://www.prisma.io/docs/guides/development-environment/environment-variables) that Prisma expects with the connection parameters provided on db creation. You'll need to add your [Github personal acccess token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for the `ecr_builder` module to be able to do the pull. Also, if you're going to deploy a codebase other than the default one, set the `REPO_URI` variable. In this case:
```